src/main/resources/org/gradle/profiler/report/report-template.html

```diff
@@ -148,12 +142,21 @@
 </div>
 </div>
 <footer>
-    <span>Gradle Profiler version {{ environment.profilerVersion }}</span>
+    <span>Gradle Profiler version {{ benchmarkResult.environment.profilerVersion }}</span>
 </footer>
 </div>
 <script>
 Array.prototype.unique = function() { return [...new Set(this)]; };
```

Comment:
The `unique` function is now unused.

Reply:
I removed it and moved the utility functions into `Array` prototype functions.